roadmap/roadmap.adoc

```diff
@@ -156,6 +158,13 @@ Model-to-model generators would be implemented as <<{refarch}.adoc#processor, Pr
 [[versions]]
 == Versions
 
+[[release-2024.1]]
+[[version-2024.1]]
+=== 2024.1
+* Specified <<{bulk}.adoc#bulk, Bulk Repository Access API>>
```

Comment:
Was that not started under 2023-1?

Reply:
2023.1 was released some time ago. It will never ever change.